controllers/performanceprofile_controller_test.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components"
 	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/kubeletconfig"
 	"github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/machineconfig"
+	pinfo "github.com/openshift-kni/performance-addon-operators/pkg/controller/performanceprofile/components/profileinfo"
```

Comment:
do we have 2 types of profile info ?  why do we need a differentiation here?

Reply:
This is just a shorthand instead of using the full ```profileinfo``` name